index.js

```diff
@@ -6,10 +6,12 @@ function createFunction(parameters, defaultType) {
     if (!isFunctionDefinition(parameters)) {
         fun = function() { return parameters; };
         fun.isFeatureConstant = true;
-        fun.isGlobalConstant = true;
+        fun.isZoomConstant = true;
 
     } else {
-        var property = parameters.property === undefined ? '$zoom' : parameters.property;
+        var zoomAndDataDependent = typeof parameters.stops[0][0] === 'object';
+        var dataDependent = zoomAndDataDependent || parameters.property !== undefined;
```

Comment:
What we've been referring to as "data-driven functions" I think we rebrand as "property functions" or "feature functions" or "feature property functions" before shipping. "Data-driven" is a more ambiguous name and potentially encompasses a larger suite of features including "feature property functions"